src/configure/configure.ts

```diff
@@ -101,12 +102,16 @@ class PipelineConfigurer {
 
         });
 
+        telemetryHelper.setCurrentStep('PostPipelineCreation');
+        // This step should be determined by the resoruce target provider (azure app service, function app, aks) type and pipelineProvider(azure pipeline vs github)
+        this.updateScmType(queuedPipeline);
```

Comment:
you should have this under try-catch. if post deployment operation fails, then user should not see any error. Warning at max. browse pipeline button should be shown